server/handlers/clients.js

```diff
@@ -0,0 +1,256 @@
+const Config = require('bcfg');
+const assert = require('bsert');
+
+const { loadClientConfigs } = require('../loadConfigs');
+
+const { configHelpers, clientFactory } = require('../utils');
+const {
+  getDefaultConfig,
+  createClientConfig,
+  createConfigsMap,
+  testConfigOptions,
+  deleteConfig,
+  getConfig
+} = configHelpers;
+
+let configsMap = null;
+
+function getClientInfo(req, res) {
+  const { logger, config } = req;
+  const clientConfigs = loadClientConfigs(config);
+  const clientInfo = {};
+
+  for (let client of clientConfigs) {
+    if (!client.str('chain'))
+      logger.warn(
+        `Client config ${client.str(
+          'id'
+        )} had no chain set, defaulting to 'bitcoin'`
+      );
+    clientInfo[client.str('id')] = {
+      chain: client.str('chain', 'bitcoin'),
+      services: {
+        node: client.bool('node', true),
+        wallet: client.bool('wallet', true),
+        multisig: client.bool('multisig', true)
+      }
+    };
+  }
+
+  return res.status(200).json(clientInfo);
+}
+
+function getDefaultClientInfo(req, res) {
+  const { config } = req;
+  const defaultClientConfig = getDefaultConfig(config);
+
+  // if there is no default config, return a 500
+  if (!defaultClientConfig)
+    return res.status(500).json({
+      error: {
+        message: `Error retrieving default client: ${defaultId}`,
+        code: 500
+      }
+    });
+
+  const defaultId = defaultClientConfig.str('id');
+  const defaultClient = {
+    id: defaultId,
+    chain: defaultClientConfig.str('chain', 'bitcoin'),
+    services: {
+      node: defaultClientConfig.bool('node', true),
```

Comment:
Default these to false and then add as part of the docs that you need to place these in the config for the app to know they exist?

Reply:
Yep! Can do. The only concern I guess is that it will likely break existing configs people are using. I will make a note for Release Notes.

Reply:
That said, thinking about having to update my own configs, I'm not sure about this solution either. If you have a normal bcoin node running, all you should really need in the config is host, network, and apiKey(s). Doing this as some sort of health check seems better to me. Would you be ok leaving this as is and doing as a separate PR so we can discuss further?

Reply:
I'm ok with leaving this as is for now but I do think that its high priority to figure out how to do this the right way